src/python/grpcio_tests/tests/unit/framework/common/__init__.py

```diff
@@ -17,17 +17,27 @@
 import socket
 
 _DEFAULT_SOCK_OPTION = socket.SO_REUSEADDR if os.name == 'nt' else socket.SO_REUSEPORT
+_UNRECOVERABLE_ERRORS = ('Address already in use',)
```

Comment:
Nit: Is there any way we can depend on error codes instead of error strings?

Reply:
Done! Also, good idea on using `in` directly.